plugins/bazaar/README.md

```diff
@@ -76,17 +76,17 @@ export const homePage = (
        <Grid container spacing={3}>
 
 +       <Grid item xs={12} md={6}>
-+         <BazaarOverviewCard variant={'latest'} limit={4} />
++         <BazaarOverviewCard order={'latest'}/>
 +       </Grid>
 
-+       <Grid item xs={12} md={6}>
-+         <BazaarOverviewCard variant={'random'} limit={4} />
++       <Grid item xs={12} md={12}>
```

Comment:
the `md` is actually redundant now

Reply:
Thank you for you comments @freben :smile:  I've learned a lot from them. Will fix these things :+1: